core/components/atoms/pills/__tests__/Pills.test.tsx

```diff
@@ -29,3 +30,32 @@ describe('Pills component', () => {
 
   testHelper(mapper, testFunc);
 });
+
+describe('Pills component', () => {
+  it('renders children', () => {
+    const { getByTestId } = render(<Pills subtle={true} appearance="secondary">{'Design System'}</Pills>);
+    expect(getByTestId('DesignSystem-Pills').textContent).toMatch('Design System');
+    expect(getByTestId('DesignSystem-Pills')).toHaveClass('Pills');
```

Comment:
Remove this test and add a test for `span` element in a different describe block.